Main.qml

```diff
@@ -307,6 +292,18 @@ ApplicationWindow {
             }
         }
 
+        Menu {
+            id: contextMenu
+            MenuItem {
+                text: qsTr("Copy all as CSV")
+                onTriggered: { appmodel.copyValidationReport(); toast.show(qsTr("Copied all results as a CSV")) }
+            }
+        }
+
+        function openContextMenu() {
+            contextMenu.open()
```

Comment:
It seems passing the `x,y` parameters is optional - it defaults to the current mouse position.